### ISSUE TITLE
Editor will not open, missing question Id, and Daily

### ISSUE THREAD

**Issue by PhilMurzynowski** [akarsh1995/leetcode-tui#6]
Hi there, clean idea!

I ran into a few issues, all of which may be stemming from one problem related to editor setup.
I am on OSX, using cookies from firefox, and use nvim as my main editor.

1. Pressing `e` my editor flashes the terminal but does nothing else. I have not been able to see or write any solution code as I cannot open an editor.
2. I wanted to rerun some questions I have previously solved to test it and got
<img width="939" alt="Pasted Graphic 2" src="https://github.com/akarsh1995/leetcode-tui/assets/26311179/d9cac3ef-ef37-4358-9365-2eafa7ed5a16">
<img width="939" alt="Pasted Graphic 3" src="https://github.com/akarsh1995/leetcode-tui/assets/26311179/a8f48190-4242-4a42-9e6f-6414e38c8dc3">
3. Finally, is this up to date w/ the daily? I think a big use case for many users is to solve the daily but I do not believe it appeared when I searched for it.

**Comment by akarsh1995** [comment]
To solve 
> I wanted to rerun some questions I have previously solved to test it and got

Please define your existing solutions_dir in the config. The solutions dir must either be empty or should have encoded named solution files. 

![image](https://github.com/akarsh1995/leetcode-tui/assets/25827628/3481be8a-c8e0-40e2-89b6-f97f66c627d1)
![image](https://github.com/akarsh1995/leetcode-tui/assets/25827628/98c101ba-b6a9-43b6-9948-d82c00a4feed)


You'll need to restart your app after defining solutions_dir.

**Comment by akarsh1995** [comment]
> Pressing e my editor flashes the terminal but does nothing else. I have not been able to see or write any solution code as I cannot open an editor.

Is it happening with every question or a few questions?


> 3. Finally, is this up to date w/ the daily? I think a big use case for many users is to solve the daily but I do not believe it appeared when I searched for it. 

I'll implement this.

**Comment by PhilMurzynowski** [comment]
> To solve
> 
> > I wanted to rerun some questions I have previously solved to test it and got
> 
> Please define your existing solutions_dir in the config. The solutions dir must either be empty or should have encoded named solution files.
> 
> ![image](https://private-user-images.githubusercontent.com/25827628/324324598-3481be8a-c8e0-40e2-89b6-f97f66c627d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3OTQ1ODIsIm5iZiI6MTcxMzc5NDI4MiwicGF0aCI6Ii8yNTgyNzYyOC8zMjQzMjQ1OTgtMzQ4MWJlOGEtYzhlMC00MGUyLTg5YjYtZjk3ZjY2YzYyN2QxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDEzNTgwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjMzgwOWNkY2ZiNDNiODJlNjI4NTdhYmU1NDgzNjg3ZmRiYzlmOTcyZGIzOWUwODVmNDg1N2ZjNWM2ZmI0N2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RxWbK7ewZ0F1dd_f0_cD_fmRHPsaD6GxQei1XhWdPDQ) ![image](https://private-user-images.githubusercontent.com/25827628/324324843-98c101ba-b6a9-43b6-9948-d82c00a4feed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3OTQ1ODIsIm5iZiI6MTcxMzc5NDI4MiwicGF0aCI6Ii8yNTgyNzYyOC8zMjQzMjQ4NDMtOThjMTAxYmEtYjZhOS00M2I2LTk5NDgtZDgyYzAwYTRmZWVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDEzNTgwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2NWZkMjY2ZjZkYjBmZDE5YmZiNGEzYTIxMzZkNDFkZDJiYzVmMGVlNTdiYzBiY2UzMTAwZGMyMjkzYjE4MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fU_J1B3WUFvGBwm6D-xgvqd1VxVT05-ab76kQdDfSXY)
> 
> You'll need to restart your app after defining solutions_dir.

Yes I forgot to mention that I had done so. I have an empty solutions dir.

**Comment by PhilMurzynowski** [comment]
> > Pressing e my editor flashes the terminal but does nothing else. I have not been able to see or write any solution code as I cannot open an editor.
> 
> Is it happening with every question or a few questions?
> 
> > 3. Finally, is this up to date w/ the daily? I think a big use case for many users is to solve the daily but I do not believe it appeared when I searched for it.
> 
> I'll implement this.

About the daily, cool!
I’m not sure about every question, but it has happened for the 5 questions I tried, a couple of which I picked randomly.

**Comment by akarsh1995** [comment]
pressing 'd' jumps you to the daily question
